src/Apps/Order2/Routes/Checkout/Components/DeliveryOptionsStep/Order2DeliveryOptionsStep.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Flex, Text } from "@artsy/palette"
+import {
+  CheckoutStepName,
+  CheckoutStepState,
+} from "Apps/Order2/Routes/Checkout/CheckoutContext/types"
+import { useCheckoutContext } from "Apps/Order2/Routes/Checkout/Hooks/useCheckoutContext"
+
+interface Order2DeliveryOptionsStepProps {}
+
+export const Order2DeliveryOptionsStep: React.FC<
+  Order2DeliveryOptionsStepProps
+> = () => {
+  const { steps } = useCheckoutContext()
+
+  const stepState = steps?.find(
+    step => step.name === CheckoutStepName.DELIVERY_OPTION,
+  )?.state
+
+  return (
+    <>
+      {stepState === CheckoutStepState.UPCOMING && (
+        <Flex flexDirection="column" backgroundColor="mono0">
+          <Box p={2} data-testid="DeliveryOptionsStep">
+            <Flex flexDirection="column">
+              <Text variant="sm-display" fontWeight="bold" color="mono100">
+                Shipping method
+              </Text>
+              <Text variant={["xs", "xs", "sm"]} color="mono60">
+                Options vary based on address and artwork size
+              </Text>
+            </Flex>
+          </Box>
+        </Flex>
+      )}
+    </>
+  )
```

Comment:
The `<>` fragment is only needed if you want to return more than a single node, like 
```tsx
<>
  <Spacer />
  <Text>I'm a special message that needs a spacer if present</Text>
</>
```

In this case for readability I might do something like
```tsx
if (stepState === CheckoutStepState.UPCOMING) {
  return <ThisWholeThing />
}

return null
```

This makes it clear when we come back to it that we only handle the one state for now.